.changelog/21342.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+agent: removed reflected cross-site scripting vulnerability
```

Comment:
Can we link to the vulnerability? IDK if we can put URLS in changelogs

Reply:
We typically do for CVE's but the scanning tool I don't think is public and didn't cite a CVE